greymatter/grocerylist/apple.cue

```diff
@@ -1,7 +1,7 @@
 package greymatter
 
 let Name = "apple"
-let AppleIngressName = "\(Name)-ingress-to-apple"
+let IngressName = "\(Name)-ingress-to-\(Name)"
 let EgressToRedisName = "\(Name)-egress-to-redis"
```

Comment:
Again, do we really need to expose Redis to end users?

Reply:
You make me think that maybe we could do something in intermediates.cue. Let me get back to you.

Reply:
I think we can move all of the redis to intermediates and in the gitops-core it should be a simple toggle:

enable_healthchecks: true

Reply:
As long as its the managed mesh redis we can hide it away. If its user managed I would say we need this

Reply:
@chrismholmes Yeah, I think a health check toggle makes sense. I also think we can fully hide the mesh Redis configs in intermediates.cue. There's no reason for the users to manage any of these configs.

Reply:
We could possibly extract the Redis configuration objects into a definition function like I did with [opa egress](https://github.com/greymatter-io/gitops-core/blob/c5d2f776e271a9c063e257b6aec5450a0811dbfa/gm/intermediates.cue#L691) and [jwt security egress](https://github.com/greymatter-io/gitops-core/blob/4dd08f735038d0c0fbc93c2325d139d168c28919/gm/intermediates.cue#L735). This would also have the added benefit of introducing that pattern to users from the get-go, allowing for familiarity when configuring more "advanced" filters.